marimo/_config/config.py

```diff
@@ -381,6 +381,19 @@ class DatasourcesConfig(TypedDict):
     auto_discover_columns: NotRequired[Union[bool, Literal["auto"]]]
 
 
+@mddoc
+@dataclass
+class SharingConfig(TypedDict):
+    """Configuration for sharing features.
+
+    **Keys.**
+
+    - `enabled`: if `False`, sharing options will be hidden from the UI
+    """
+
+    enabled: bool
+
```

Comment:
I think this could be more granular:

* html
* wasm

both booleans. wasm doesn't leak (we don't save code).